test/queries.js

```diff
@@ -446,3 +446,42 @@ test('search documents using text index', async t => {
 	t.is(posts[0].get('title'), 'San Francisco');
 	t.is(posts[1].get('title'), 'San Fran');
 });
+
+test('get distinct', async t => {
+	try {
```

Comment:
What indentation is missing? also npm test passed checking style guide defined, is there any thing extra i need to check?